packages/react-component-library/src/components/ProgressBar/fixProgressValue.ts

```diff
@@ -0,0 +1,21 @@
+import logger from '../../utils/logger'
+
+export function fixProgressValue(value: number, defaultValue = 0): number {
+  const num = Number(value)
```

Comment:
This conversion to `Number` appears to be redundant and using `num` would have not been the best name.

Reply:
Not sure I follow you.
I did this because `Number.isNaN('dodgy value from an api')` is `false`

Reply:
Ah okay, I slightly misunderstood the code. I think something like `numberValue` would be better name.